doc/RELEASE-NOTES.md

```diff
@@ -0,0 +1,190 @@
+
+Along with the delivery of the "Tool Meister" work, the notion of tool registration has changed significantly, where tools are now only registered locally on the host were `pbench-register-tool` and pbench-register-tool-set` are invoked.  As a result of this change, the following behavioral changes follow:
+
+ * The process of registering tools on local or remote hosts no longer validates that those tools are available during tool registration
+ * All tools must be registered after installing `v0.70.0-beta`; tools registered on a controller with a version of the `pbench-agent` prior to `v0.70.0`, whether local or remote, will be ignored
+
+For both the Pbench Agent and the Server we have removed the use of the Software Collections Library (SCL) in order to provide a Python 3 environment for RHEL 7 & CentOS 7 distributions.  We now rely on the Python 3 provided with RHEL 7.7 and later.
+
+We did not bump the "major" release version number with these changes because we still don't consider all the necessary functionality in place for such a major version bump.
+
+Installation
+====
+There are ansible playbooks to install the `pbench-agent` and the pieces needed (key and config files) to be able to send results to a server.
+
+There are no other installation changes in this release: see the [Getting Started Guide](https://distributed-system-analysis.github.io/pbench/start.html) for how to install or update.
+
+After installation or update, you should have version `0.70.0-1gXXXXXXXXX` of the `pbench-agent` RPM installed.
+
+Agent
+====
+In addition to the major changes described above for this release, the following significant changes for the agent are also worth calling out specifically:
+
+ * The `pbench-cleanup` utility command is deprecated and will be removed in a subsequent release (see PR #1828)
+
+ * The release takes in the latest code from stockpile, including support for GPU data collection via the [Nvidia SMI](https://github.com/cloud-bulldozer/stockpile/tree/master/roles/nvidia_smi) role
+
+ * The new `dcgm` tool requires Python 2, an Nvidia based install which might conflict with the Pbench Agent's Python 3 operational requirement in some cases
+
+Server
+====
+Initial support for moving results to the pbench-server via HTTP `PUT` operations.
+
+Web Server
+====
+There are no changes and no new `web-server` RPMs have been produced.
+
+Pbench Dashboard
+====
+The development of the dashboard is not tracked in these release notes. The dashboard has been moved into its own [git repo](https://github.com/distributed-system-analysis/pbench-dashboard).
+
+ChangeLog
```

Comment:
Building an RPM _could_ automatically update a release notes template with the ChangeLog and installing it somewhere. Hmm.

Reply:
_**Could**_.  =)

Can we work on this for the next release?

I filed issue #1965 to track.